src/assets/scss/components/playground-configuration.scss

```diff
@@ -136,6 +136,15 @@
     padding: 1rem;
 }
 
+.playground__config-enable_all_rules {
+    display: flex;
+    margin: 0.5rem 0;
+}
+
+.enable_all_rules-checkbox {
+    margin: 0 0.4rem 0.2rem 0;
+}
```

Comment:
```suggestion
```

Now we're not using this class anymore.

Reply:
Removed it thanks. Just to confirm pushed a separate commit instead of committing suggestion